components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -1804,5 +1804,6 @@ with se_tstep, dt_remap_factor, dt_tracer_factor set to -1
 <so4_sz_thresh_icenuc phys="default"> 0.080e-6   </so4_sz_thresh_icenuc>
 <micro_mg_berg_eff_factor phys="default" clubb_sgs="1"> 0.7D0      </micro_mg_berg_eff_factor>
 <cldfrc_dp1  phys="default"> 0.018D0    </cldfrc_dp1>
+<cosp_ncolumns phys="default" hgrid="ne1024np4"> 10 </cosp_ncolumns>
```

Comment:
I'd rather delete `hgrid="ne1024np4"` in the line above so we always use 10 subcolumns unless we make an explicit change. Otherwise I'm sure we're going to think we're doing the same thing at ne512 or 256 and wonder why we get different timings or results...

Reply:
Thanks for chiming in, @PeterCaldwell . The code changes largely to allow a resolution dependent of the parameter (and to preserve existing cosp tests). It makes senses at least for SCREAM simulations, we want it to be 10.

Reply:
@wlin7 actually no tests use COSP without `cosplite`, so this would not affect any existing tests. It appears that COSP is never used in E3SM without setting `cosp_ncolumns = 10`.

Reply:
Thanks for checking on the existing cost tests, @brhillman . I remove the explicit grid dependence.

Actually, in our SCREAM simulations so far, with cosp_lisccp_sim, there is no internal mechanism to reset the hardwired default of 50. If setting cosp_isccp = .true. instead, it would.